client/web/src/enterprise/site-admin/dotcom/productSubscriptions/SiteAdminProductLicenseNode.tsx

```diff
@@ -89,90 +119,92 @@ export const SiteAdminProductLicenseNode: React.FunctionComponent<
                             <LoaderButton
                                 className="ml-auto"
                                 variant="danger"
                                 label="Revoke"
                                 onClick={onRevoke}
-                                loading={loading}
+                                loading={isRevokeLoading}
                             />
                         )}
                     </CollapseHeader>
                     <div />
                     <CollapsePanel className="mt-4">
                         <div className="d-flex">
                             <Label>License Key ID</Label>
-                            <Text className="ml-3">{uuid}</Text>
+                            <Text className="ml-3">
+                                <span className="text-monospace">{node.id}</span>
+                            </Text>
                         </div>
                         <div className="d-flex">
                             <Label>Key Version</Label>
-                            <Text className="ml-3">{node.version}</Text>
+                            <Text className="ml-3">{licenseKey.infoVersion}</Text>
                         </div>
-                        {node.version > 1 && (
+                        {licenseKey.infoVersion > 1 && (
                             <>
-                                <div className="d-flex">
-                                    <Label>Site ID</Label>
-                                    <Text className="ml-3">
-                                        {node.siteID ?? <span className="text-muted">Unused</span>}
-                                    </Text>
-                                </div>
-                                <div className="d-flex">
-                                    <Label>Salesforce Subscription ID</Label>
-                                    <Text className="ml-3">
-                                        {node.info?.salesforceSubscriptionID ?? (
-                                            <span className="text-muted">Unused</span>
-                                        )}
-                                    </Text>
-                                </div>
                                 <div className="d-flex">
                                     <Label>Salesforce Opportunity ID</Label>
                                     <Text className="ml-3">
-                                        {node.info?.salesforceOpportunityID ?? (
-                                            <span className="text-muted">Unused</span>
+                                        {info?.salesforceOpportunityId ? (
+                                            <span className="text-monospace">{info?.salesforceOpportunityId}</span>
+                                        ) : (
+                                            <span className="text-muted">Uknown</span>
```

Comment:
You worked with Joe for too long 😬 
```suggestion
                                            <span className="text-muted">Unknown</span>
```

Reply:
haha, fixed!